scss/ouds-web-utilities.scss

```diff
@@ -3,6 +3,9 @@
 
 // Configuration
 @import "functions";
+@import "tokens/raw";
+@import "tokens/semantic";
+@import "tokens/component";
```

Comment:
`@import "tokens/component";` is not needed to compile

Reply:
Since the PR is for the token architecture and documentation, should'nt we have this imported even if the file is empty ? You think it should be added only when we have our first component ? Do we completely delete the tokens/component file if it is not used anywhere ?

Reply:
I'd keep it for `ouds-web.scss` since it will be used in the end for sure. For the other `ouds-web-*.scss` files, it's not mentioned in the documentation, so I'd import only the necessary files.